README.md

```diff
@@ -138,6 +139,16 @@ Adjusts the defined matcher paths for caddy. You might want to adjust those to a
 kellerkinder.staticFilePaths = "myCustomFolder/*";
 ```
 
+### kellerkinder.fallbackMediaUrl
+This url is called when the local media does not exist. Best for CDN purposes without downloading them.
```

Comment:
Maybe like this. Is more understandable to me.
```suggestion
Proxy URL for media. Best for CDN purposes without downloading them.
```

Reply:
I don't see this as better wording, while this is no proxy 🤔 It's an address where the client is redirected to.

Reply:
Then how about "Fallback redirect URL for media"? 🤔